signing/transaction-manager.js

```diff
@@ -56,21 +57,40 @@ export async function createSignBroadcast({
     gas: String(transactionData.gasEstimate),
   }
 
-  const client = new SigningCosmosClient(
-    network.apiURL,
-    senderAddress,
-    signer,
-    undefined,
-    undefined,
-    BroadcastMode.Async
-  )
-  const broadcastResult = await client.signAndBroadcast(
+  const signDoc = makeSignDoc(
     [].concat(messages),
     fee,
-    transactionData.memo
+    chainId,
+    transactionData.memo,
+    accountInfo.accountNumber,
+    accountInfo.sequence
   )
+  const { signed, signature } = await signer.sign(senderAddress, signDoc)
+  const signedTx = makeStdTx(signed, signature)
+
+  const broadcastBody = {
+    tx: signedTx,
+    mode: 'sync',
+  }
+  // TODO use axios?
+  const broadcastResult = await fetch(
+    `https://api.allorigins.win/get?url=http://34.123.30.100:1317/txs`,
+    {
+      method: 'POST',
+      body: JSON.stringify(broadcastBody),
+    }
+  ).then((res) => res.json())
   assertIsBroadcastTxSuccess(broadcastResult)
 
+  if (
+    !assertIsBroadcastTxSuccess(broadcastResult) &&
+    broadcastResult.contents
```

Comment:
What is assertion fails and content is not available? I guess this check needs to move inside the if block